### PR TITLE
Merge upstream from Node Starter App 6.0.0

### PR DESCRIPTION
**Pull request by dsamojlenko**
Merging in the latest from node-starter-app, version 6.0.0.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -64,21 +64,16 @@ typings/
...
-
-
-public/js/dist
 
 # Visual Studio Code
-.vscode
\ No newline at end of file
+.vscode
+
+credentials
```
Comment (CalvinRodo) [inline .gitignore:79]: Why is credentials multiple times?

Comment (CalvinRodo) [reply .gitignore:79]: is this an upstream merge or something you added?

**Comment by CalvinRodo** [comment]
The heroku deployment isn't working.

**Comment by timarney** [comment]
Patched upstream - https://github.com/cds-snc/node-starter-app/commit/40e49fdf1689f7642a0986918edb67b0d7200cff

File: .gitignore (hunk 1 above)
Comment (dsamojlenko) [reply .gitignore:79]: That's upstream... looks like it's been there at least since the previous release

**Review by jeffmaher — APPROVED** [review]
LGTM 🍕

**Pull request merged**